pyproject.toml

```
@@ -12,7 +12,7 @@ description = "A fork of the GYP build system for use in the Node.js projects"
 readme = "README.md"
 license = { file="LICENSE" }
 requires-python = ">=3.8"
-dependencies = ["packaging>=24.0"]
+dependencies = ["setuptools>=69.5.1"]
```

I believe `packaging` should not be removed as installing `setuptools` doesn't come with `packaging`.

```suggestion
dependencies = ["packaging>=24.0", "setuptools>=69.5.1"]
```

---

Downloading setuptools-69.5.1-py3-none-any.whl from https://pypi.org/project/setuptools/#files and unarchiving it, I didn't find that `packaing` is included in `setuptools`.

---

https://github.com/search?q=repo%3Apypa%2Fsetuptools%20packaging&type=code

---

Yes, `packaging` is vendored in setuptools. But as far as I could tell, `packaging` in setuptools is not exposed in `site-packages` with setuptools. So it would need to import like `from setuptools._vendor.packaging.version import Version`.

---

@cclauss I created a github action to verify this change: https://github.com/legendecas/gyp-next/actions/runs/9127187853